Resources/Private/Templates/CompactView/Index.html

```diff
@@ -1,19 +1,17 @@
-<link rel="stylesheet" type="text/css"
-      href="https://d8ejoa1fys2rk.cloudfront.net/modules/compactview/includes/css/compactview_global-1.0.1.css">
-
 <div id="bynder-compactview"
-     data-language="{language}"
-     data-button="Pick a file from Bynder"
-     data-folder="bynder-compactview"
-     data-zindex="300"
-     data-collections="false"
-     data-header="false"
+     data-assetTypes="{assetTypes}"
      data-autoload="true"
-     data-fullScreen="true"
+     data-collections="true"
```

Comment:
Does that work selecting collections? Or does this only allow to browse collections?

Reply:
For me, it only allowed the quick filter of collections (in the bottom)

Reply:
ok, so you still get single assets. It only enables you to browse the collections?